modules/roxotAnalyticsAdapter.js

```diff
@@ -37,12 +37,16 @@ function buildEventStack() {
 
 function send(eventType, data, sendDataType) {
   let fullUrl = url + '?publisherIds[]=' + initOptions.publisherIds.join('&publisherIds[]=') + '&host=' + window.location.hostname;
-
-  ajax(
-    fullUrl,
-    (result) => utils.logInfo('Event ' + eventType + ' sent ' + sendDataType + ' to roxot prebid analytic with result' + result),
-    JSON.stringify(data)
-  );
+  let xhr = new XMLHttpRequest();
```

Comment:
Fair point. We should allow analytics adapters to override the timeout. For now this is fine.